pmm/aws-staging-start.groovy

```diff
@@ -477,9 +477,9 @@ pipeline {
                                     else
                                         pmm-agent setup --config-file=$PWD/pmm2-client/config/pmm-agent.yaml --server-address=\$IP:443 --server-insecure-tls --server-username=admin --server-password=admin \$IP
                                     fi
-                                    sleep 10
+                                    sudo sleep 10
```

Comment:
why just sleep doesn't work here?

Reply:
that is the main question. I assume PATH was missed somehow